twitter-borrowbot/twitterbot.py

```diff
@@ -93,30 +114,54 @@ def internal_error(cls, mention):
         )
 
 def get_last_seen_id():
-    with open(STATE_FILE, 'r') as fin:
-        return int(fin.read().strip())
-
+    try:
+        with open(STATE_FILE, 'r') as fin:
+            last_seen_id = fin.read().strip()
+    except Exception as e:
+        raise FileIOError(file=STATE_FILE, error=e)
```

Comment:
I don't think you have to worry about this, the STATE_FILE is being opened using `with` and if an error occurs, `open` will emit a file IO exception